docs/intro/glossary.md

```diff
@@ -0,0 +1,255 @@
+
+Same as [net consumption](#net-consumption).
+
+### Residual Consumption
+
+In [microgrid](#microgrid) context sometimes used as the remaining difference between the [net consumption](#net-consumption) and the [battery](#battery) power, i.e. what we define as [grid power](#grid_power).
+
+### Residual Load
+
+Same as [residual consumption](#residual-consumption).
+
+### State of Charge
+
+The level of charge of a [battery](#battery) relative to its capacity, expressed in percentage points. Calculated as the ratio between the remaining energy in the battery at a given time and the maximum possible energy under similar health conditions. [Source](https://epicpower.es/wp-content/uploads/2020/08/AN028_SoC-SoH-SoP-definitions_v3.pdf)
+
+### State of Power
+
+The ratio of peak power to nominal power. Peak power is the maximum power that can be sustained for a specific duration without violating preset operational design limits on battery voltage, SsC, power, or current.
+
+This indicator is crucial to ensure that charge or discharge power remains within specific limits, optimizing the battery's lifespan. It is particularly useful in peak power applications to define battery conditions for substantial charges or discharges.
+
+The state of power depends on the state of charge, battery capacity, initial characteristics, chemistry, and battery voltage. [Source](https://epicpower.es/wp-content/uploads/2020/08/AN028_SoC-SoH-SoP-definitions_v3.pdf)
+
+## Microgrid Power Terminology
+
+Within the SDK, the terminology used for [measurements](#measurement), such as the power of individual [microgrid](#microgrid) [components](#component), follow the pattern `{component}_{quantity}` for total values and `{component}_{consumption,production}_{quantity}` for clipped values.
+
+Valid components include `grid`, `battery`, `ev_charger`, `consumer`, and active components `pv`, `chp`, and `wind`.
+
+The SDK provides the following power metrics IDs.
+
+### `grid_power`
+
+Can be positive (indicating consumption from the grid) or negative (indicating production into the grid).
+
+### `{battery,ev_charger}_power`
+
+Can be positive (indicating [battery](#battery) charging) or negative (indicating battery discharging). This also applies to [EV charging](#ev-charger) stations.
+
+### `{pv,chp,wind}_power`
+
+Pertaining to active [components](#component), this represents power generated from local (renewable) sources. It is negative when electricity is generated, otherwise zero or positive in case of self-consumption. Alternative terms include generation, supply, and source.
+
+### `consumer_power`
+
+Aggregates [components](#component) not covered above. Under typical circumstances, this corresponds to the site's [gross consumption](#gross-consumption), excluding active parts and the [battery](#battery). The term 'consumer' implies that this component predominantly consumes electric power. It can support negative values for exotic site topologies or short-term effects.
```

Comment:
`Aggregates` => `Covers`? (aggregate could mean something else if the reader is a data scientist, e.g.).

Reply:
Nitpick: `short-term effects.` => `short durations.`